drivers/health_qa_factory/app/models/health_qa_factory/factory.rb

```diff
@@ -19,6 +19,16 @@ def complete?
       careplan_completed_qa.present?
     end
 
+    def missing_components?
+      ! [
+        hrsn_screening_qa.present?,
+        careplan_development_qa.present?,
+        ca_completed_qa.present?,
+        careplan_development_qa.present?,
+        careplan_completed_qa.present?,
+      ].all?
+    end
```

Comment:
Maybe:
```suggestion
    def missing_components?
      [
        hrsn_screening_qa.present?,
        careplan_development_qa.present?,
        ca_completed_qa.present?,
        careplan_development_qa.present?,
        careplan_completed_qa.present?,
      ].any?(false)
    end
```
just for readability?

Reply:
```
irb(main):011:0> [true, true].any?(false)
=> false
irb(main):012:0> [true, false].any?(false)
=> true
irb(main):013:0> ! [true, false].all?
=> true
irb(main):014:0> ! [true, true].all?
=> false
```